srttools/monitor.py

```diff
@@ -14,12 +14,23 @@
 except ImportError:
     PatternMatchingEventHandler = object
     HAS_WATCHDOG = False
+
 import warnings
 import glob
 from threading import Timer, Thread
-from multiprocessing import Process, Queue
-from queue import Empty
-from http.server import HTTPServer, SimpleHTTPRequestHandler, HTTPStatus
+from multiprocessing import Process, Queue, cpu_count
+
+try:
+    from queue import Empty
+    HAS_QUEUE = True
+except ImportError:
+    HAS_QUEUE = False
+
+try:
+    from http.server import HTTPServer, SimpleHTTPRequestHandler, HTTPStatus
+    HAS_HTTP = True
+except ImportError:
```

Comment:
This can be replaced with these two lines:
from BaseHTTPServer import HTTPServer
from SimpleHTTPServer import SimpleHTTPRequestHandler
HTTPStatus can be completely removed, replace `HTTPStatus.NOT_FOUND` in line 152 with just 404 (no quotes are needed)

Reply:
If you want to add more statuses, it's convenient to use the class. I just created a dummy class in the Py2 case (not sure if it will ever be useful, anyway. )